apps/dashboard/app/helpers/active_jobs_helper.rb

```diff
@@ -58,4 +58,36 @@ def has_ganglia(host)
   def has_grafana(host)
     OODClusters[host].try { |cluster| cluster.custom_allow?(:grafana) } || false
   end
+
+  def status_label(status)
```

Comment:
This was originally defined in [application_helper.rb](https://github.com/OSC/ondemand/blob/a0204bb9a9cbda12fd00d38c8eadfac73283bbb3/apps/activejobs/app/helpers/application_helper.rb). A file which could not be ported directly, so all of it's contents (which is only this method) are added here.